docs/auth/README.md

```diff
@@ -109,17 +164,20 @@ jexiaClient().init({
 
 // Sign in with the user created in your Jexia project
 ums.signIn({  
-  email: 'user@jexia.com',  
+  email: 'admin@x.com',  
   password: 'secret-password'
-});  
+}).subscribe(
+  data=>{ ...do future selects... },
+  error=>{... error handle...}
+);  
 ```
 </template>
 <template v-slot:bash>
 
 ``` bash
 # Environment variables to be set
 export PROJECT_ID=<project_id>
-export TEST_USER=<user_here>
+export TEST_USER=admin@x.com
```

Comment:
Why did you change this? All other variables are template why do we need to have fake email address here?

Reply:
It was a minute of creativity...